### PR TITLE
[REVIEW] Python v3.7 deprecation notice

### PR DESCRIPTION
**Pull request by Salonijain27**

### CODE REVIEW

**Review by fondaing — COMMENTED** [review]
LGTM

**Comment by aravenel** [comment]
Can we make language/structure mirror previous deprecations/EOLs we have done? E.g.:

https://docs.rapids.ai/notices/rsn0005/
https://docs.rapids.ai/notices/rsn0006/

Just want to keep process the same where we can.

File: _notices/rsn0013.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## Overview
+
+`Python 3.7` will be deprecated in our upcoming release of `v22.02`
+scheduled for February 2022. After the `v22.02` release, users should expect 
+that support for `Python 3.7` may be dropped at any point.
+
+## Impact
+
+Users should plan to move to `Python 3.8` as soon as possible. Future
+support of additinal `Python 3.9` version is being worked on for upcoming
```
Comment (aravenel) [inline _notices/rsn0013.md:36]:
```suggestion
support of additional `Python 3.9` version is being worked on for upcoming

```

Comment (aravenel) [inline _notices/rsn0013.md:36]:
```suggestion
support of additional `Python 3.9` version is being worked on for upcoming

```

**Review by aravenel — APPROVED** [review]
LGTM

**Review by fondaing — APPROVED** [review]
LGTM

**Pull request merged**